libs/blocks/text/text.css

```diff
@@ -158,6 +223,18 @@
   line-height: 0em;
 }
 
+.text-block .image picture {
+  display: flex;
+}
+
+.text-block.center .image picture {
+  justify-content: center;
+}
+
+.text-block.right .image picture {
```

Comment:
I was looking at this declaration and noticed the `.text-block.right` class doesn't position the text properly on RTL locales.

**RTL**
<img width="1004" alt="Screenshot 2024-09-05 at 14 24 49" src="https://github.com/user-attachments/assets/fb52b41c-a15e-42e6-9c19-c91481339831">

**RTL with `right` class applied to text-block**
<img width="990" alt="Screenshot 2024-09-05 at 14 24 59" src="https://github.com/user-attachments/assets/8bf509d0-e596-48af-9705-99e1cb01d12c">